internal/auth/jwt.go

```diff
@@ -1,6 +1,19 @@
 // Package auth provides the logic for working with JWT tokens.
 package auth
 
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+)
+
+const (
+	ACTimeout = 12 * time.Hour
+	RTTimeout = 48 * time.Hour
```

Comment:
Can we give this more descriptive names?

Reply:
> Can we give this more descriptive names?

ok, I will describe the constants more detail